Stages.md

```diff
@@ -0,0 +1,125 @@
+# Stages - a tool for encouraging attention to incubation features
```

Comment:
Not sure if a policy needs a subtitle in this way; it would feel a bit out of place among the others on https://whatwg.org/policies I think. Maybe just `# Stages` would work.

Reply:
Fixed in https://github.com/whatwg/sg/pull/222/commits/dd195992df4f01dbd374454f6642065ec8a2bbe9.